api/src/main/java/org/pmiops/workbench/mail/MailService.java

```diff
@@ -3,8 +3,13 @@
 import javax.mail.Message;
 import javax.mail.MessagingException;
 
+import com.google.api.services.admin.directory.model.User;
+import org.pmiops.workbench.mandrill.model.MandrillMessage;
+import org.pmiops.workbench.mandrill.model.MandrillMessageStatuses;
+
 public interface MailService {
 
     void send(Message msg) throws MessagingException;
 
+    MandrillMessageStatuses sendEmail(String contactEmail, String password, User user) throws MessagingException;
```

Comment:
I think this is too-generic a method name. Under the hood, it sends out what will be the welcome to AoU email. There are a couple ways of doing this. One way (and what we do more often in this codebase) is to make the method name very reflective of what it does, i.e. `sendWelcomeEmail`. 

Another approach is to bundle up the arguments into a DTO object that has more implementation-specific information. Something like
```
    public enum Type {WELCOME}
    class MessageType {
        String contactEmail;
        String password; 
        User user;
        Type type;
    }
```
And then the implementation of `sendEmail` can look at the type and do something more type specific. As more email types are defined, we can implement them one at a time.